cmd/pmm-managed-starlark/main.go

```diff
@@ -127,7 +127,7 @@ func runChecks(l *logrus.Entry, data *checks.StarlarkScriptData) ([]check.Result
 		return nil, errors.Wrap(err, "error unmarshalling query result")
 	}
 
-	results, err := env.Run(data.Name, input, l.Debugln)
+	results, err := env.Run(data.Name, input, nil, l.Debugln)
```

Comment:
for now we dont have any additional functions implemented (will be done in the next ticket SAAS-280) so we pass a nil map

Reply:
Let's pass our two version 1 predefined functions via context to be able to test it